frappe/www/login.py

```diff
@@ -143,3 +145,49 @@ def login_via_token(login_token):
 	redirect_post_login(
 		desk_user=frappe.db.get_value("User", frappe.session.user, "user_type") == "System User"
 	)
+
+
+@frappe.whitelist(allow_guest=True)
+def send_login_link(email: str, subject: str | None = None):
+	if not frappe.db.exists("User", email):
+		frappe.throw(
+			_("User with email address {0} does not exist").format(email), frappe.DoesNotExistError
+		)
+
+	key = frappe.generate_hash("Login Link", 20)
+	minutes = 10
+	frappe.cache().set_value(f"one_time_login_key:{key}", email, expires_in_sec=minutes * 60)
+
+	link = get_url(f"/api/method/frappe.www.login.login_via_key?key={key}")
+
+	app_name = (
+		frappe.get_website_settings("app_name") or frappe.get_system_settings("app_name") or _("Frappe")
+	)
+
+	subject = subject or _("Login To {0}").format(app_name)
+
+	frappe.sendmail(
+		subject=subject,
+		recipients=email,
+		template="passwordless_login",
+		args={"link": link, "minutes": minutes, "app_name": app_name},
```

Comment:
```suggestion
		args={"link": link, "minutes": minutes, "app_name": app_name},
		now=True,
```
@ankush This will not cause any problems, right?

Reply:
`now=True`? Should be fine :+1: 

Lets rate limit this by default? Doubt anyone will need this more than 5 times in an hour. 

```
@rate_limit(limit=5, seconds=24 * 60 * 60, methods=["POST"])
```